src/knowledge_sources/SegmentationKS.m

```diff
@@ -178,7 +180,7 @@
             % Execute KS if a sufficient amount of data for one block has
             % been gathered
             bExecute = (obj.blackboard.currentSoundTimeIdx - ...
-                obj.lastExecutionTime_s) >= obj.blockSize;
+                obj.lastExecutionTime_s) >= 0.1;

```

Comment:
@chrschy, I lowered the threshold to allow more frequent executions of the `SegmentationKS`. The orignal threshold was blockSize, which meant that the segmentation was being done on non-overlapping blocks.
Do you think we should add a parameter to set this threshold at instantiation time?